main.go

```diff
@@ -99,6 +102,7 @@ func main() {
 	flag.StringVar(&namespace, "namespace", "aws-load-balancer-operator", "The namespace where operands should be installed")
 	flag.StringVar(&image, "image", "quay.io/aws-load-balancer-operator/aws-load-balancer-controller:latest", "The image to be used for the operand")
 	flag.StringVar(&trustedCAConfigMapName, "trusted-ca-configmap", "", "The name of the config map containing TLS CA(s) which should be trusted by the controller's containers. PEM encoded file under \"ca-bundle.crt\" key is expected.")
+	flag.BoolVar(&webhookDisableHTTP2, "webhook-disable-http2", false, "Disable HTTP2 for the webhook server.")
```

Comment:
https://github.com/openshift/external-dns-operator/pull/204#discussion_r1388577353

```suggestion
	flag.BoolVar(&webhookDisableHTTP2, "webhook-disable-http2", false, "Disable HTTP/2 for the webhook server.")
```

Reply:
Done.